analytics-datastore-clickhouse/docker-compose.dev.yml

```diff
@@ -4,8 +4,8 @@ services:
   analytics-datastore-clickhouse:
     ports:
       - target: 8123
-        published: 8124
+        published: 8123
         mode: host
       - target: 9000
-        published: 9001
+        published: 9000
```

Comment:
Thank you, been meaning to do this too. This might have implications in other packages. Maybe a quick search will be able to tell if this is the case or not.